index.html

```diff
@@ -229,22 +229,20 @@ <h5>Skills</h5>
                 <hr/>
 			</div>
     </section>
-	
-	<!-- the contact form's section -->
-<section class="form">
-  <div class="contact-sec">
-    <h2>Contact me</h2>
-     <p>If you have an application you <br> are interested in developing,a <br> feature that you need built or a <br> project that needs coding.I'd <br> love to help with it.</p> 
-  </div>
-<form action="https://formspree.io/f/mzbowdel" method="post">
-  <ul>
-    <li><input type="text" class="name" id="name" maxlength="30" placeholder="Tshepo Yusuf" name="name" required></li>
-    <li><input type="email" class="email"  placeholder="tshepomakama45@gmail.com" id="email" name="email" required></li>
-    <li><textarea class="msg"  id="msg" maxlength="500" placeholder="Write your message here." name="message" required></textarea></li>
-    <li><button type="submit" class="btn" id="btn">Get in touch</button></li>
-  </ul>
-</form>
-</section>
-	
+
+    <section id="contact">
+        <div class="contact_detail">
+            <h2 class="head">Contact me</h2>
+            <p class="detail">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                I’d love to help with it!
+            </p>
+        </div>
+        <form class="form" action="https://formspree.io/f/mayzgvgj" method="POST">
+            <input class="input" type="text" maxlength="30" name="text" required placeholder="Enter your name">
```

Comment:
- [ ] You have done great so far, :+1:  but there are a few UI issues you need to fix before moving to the next milestone,
- - According to the project requirement, you are meant to replicate the given design as much as you can, Currently, there is no spacing between your text and the first input field, the spacing is present in the Figma design, I'll suggest you implement this :+1: :100: 
- - Your Current UI:
![Screenshot from 2023-06-13 14-39-20](https://github.com/BilalLiaquat7/Portfolio/assets/107632780/b6a4f354-6517-43e6-b0fe-c6948a696a78)

- - Figma design:
![Selection_061](https://github.com/BilalLiaquat7/Portfolio/assets/107632780/deb914aa-5c32-4900-8d0c-4e222c20c6b2)